sigh/lib/sigh.rb

```diff
@@ -2,6 +2,7 @@
 require 'sigh/resign'
 require 'sigh/manager'
 require 'sigh/options'
+require 'sigh/local_manage'
```

Comment:
Up until now this was imported on https://github.com/fastlane/fastlane/pull/4579/files#diff-1da9a235513ffba6a58c0e444473ad29L34. Instead it's much cleaner to import it in `sigh.rb` with all the other imports.